src/easylink/rule.py

```diff
@@ -182,15 +182,15 @@ def _build_shell_cmd(self) -> str:
         # TODO [MIC-5787]: handle multiple wildcards, e.g.
         #   output_paths = ",".join(self.output)
         #   wildcards_subdir = "/".join([f"{{wildcards.{wc}}}" for wc in self.wildcards])
-        #   and then in shell cmd: export DUMMY_CONTAINER_OUTPUT_PATHS={output_paths}/{wildcards_subdir}
+        #   and then in shell cmd: export OUTPUT_PATHS={output_paths}/{wildcards_subdir}
```

Comment:
These changes in this file are the primary motivator of this PR. Because the snakemake rules were hardcoding env vars as being "DUMMY_CONTAINER_..." that means all implementations scripts had to use those prefixes as well.

Note that throughout this PR there are still plenty of dummy_container references, but they're now limited to strictly the actual dummy pipeline (I thought about just removing ALL "dummy_container" prefixes but that doesn't seem all that important.)